cmd/containerd-shim-runhcs-v1/exec_hcs.go

```diff
@@ -22,18 +21,6 @@ import (
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 	"go.opencensus.io/trace"
-	"golang.org/x/sys/windows"
-)
-
-const (
-	// processStopTimeout is the amount of time after signaling the process with
-	// a signal expected to kill the process that the exec must wait before
-	// forcibly terminating the process.
-	//
-	// For example, sending a SIGKILL is expected to kill a process. If the
-	// process does not stop within `processStopTimeout` we will forcibly
-	// terminate the process without a signal.
-	processStopTimeout = time.Second * 5
```

Comment:
If this was unused code, should we instead look into why we had all of this jotted up and never actually utilized it?

Reply:
Justin made a change [here](https://github.com/microsoft/hcsshim/commit/3d974573138bbccc5177abc2a943462c1fb21ffa) that made use of it and later removed the change [here[(https://github.com/microsoft/hcsshim/commit/0bbe10215f4c337cd5a404f3fa31da269e26de37). I think it's okay to remove. (This was also like two years ago :P)

Reply:
Sure